processes/security_team_membership_policy.md

```diff
@@ -0,0 +1,28 @@
+# Node.js Security Team Membership Policy
+
+The Node.js Security Team (@nodejs/security) has
+access to security-sensitive issues and patches that aren't approriate for
+public availability.
+
+> Note: Due to implementation details and a bit of human error, the list of
+> people with access to the security patches isn't 100% in line with the
+> @nodejs/security membership. See the [membership list](./security_team_members.md)
+> for details. This situation is intended to be corrected in the near future.
+
+This isn't to be confused with the security triage team, which is currently
+defined by the [`security@nodejs.org` email alias](https://github.com/nodejs/email/blob/master/iojs.org/aliases.json).
+Inclusion in that group is on a volunteer basis, upon approval by the Technical
+Steering Committee (ie. @nodejs/TSC).
+
+The policy for inclusion on the @nodejs/security is as follows:
+
+1. All members of the Technical Steering Committee (i.e. @nodejs/TSC) are
```

Comment:
Nit: remove `i.e.`

Reply:
Do we need the @ reference here too? It was just defined a couple lines up.